cypress/integration/Admin.feature

```diff
@@ -1,8 +1,19 @@
-Feature: Admin 
- 
+Feature: Admin
```

Comment:
```suggestion
@Admin
Feature: Admin
```
Think it would be good if we could add tags so we can identify the area of the service we're testing.